Added support for dynamically setting tfplan and tfvars file names wh… (#93)
hmcts/cnp-azuredevops-libraries: README.md

    │   │    └── *.tfvars
    ├── azure_pipeline.yaml
    ├── .terraform-version                                 # terraform version (read by tfenv)

> ### Multi Regions:  

Reviewer: ```suggestion
### Multiple regions
```

> Some repos have separate `tfvars` files based on region i.e. `UK South` or `UK West` and having all enviroment

Reviewer: ```suggestion
Some repositories have separate `tfvars` files based on region i.e. `UK South` or `UK West` and having all enviroment
```

variables in a single `tfvars` file may not meet the requirement.
If you need to use a different set of variables for your build then you can pass the `multiRegion` parameter and

> set his as `true`, the default is `false`.

Reviewer: ```suggestion
set this as `true`, the default is `false`.
```

> Using multi region requires the below environments folder structure for the build repository.

Reviewer: ```suggestion
Using multiple region support requires the below environments folder structure for the build repository.
```
